I'm not sure what I need to do. I tried two things: 
- Modified the now.json in the [mars-example](https://github.com/frontity/frontity/blob/dev/examples/mars-theme-example/now.json) project, but then I get this error:
```
Error! You defined 1 build that did not match any source files (please ensure they are NOT defined in .vercelignore):
- {"src":"package.json","use":"@frontity/now"}
```
- Created a fresh frontity project with `npx frontity create`, created a now.json file, like [this](https://github.com/frontity/now-builder#before-deploying), and modifed as you suggested.
Then I get: `Error! Build failed`:

```
13:59:36.992  Retrieving list of deployment files...
13:59:37.164  Downloading 24 deployment files...
13:59:39.436  Installing build runtime...
13:59:41.825  Build runtime installed: 2388.764ms
13:59:42.108  Error: Cannot find module '/vercel/4c23e98f/.build-utils/.builder/node_modules/@frontity/now/dist/index.js'. Please verify that the package.json has a valid "main" entry
13:59:42.108      at tryPackage (internal/modules/cjs/loader.js:297:19)
13:59:42.108      at Function.Module._findPath (internal/modules/cjs/loader.js:528:18)
13:59:42.108      at Function.Module._resolveFilename (internal/modules/cjs/loader.js:780:27)
13:59:42.108      at Function.Module._load (internal/modules/cjs/loader.js:686:27)
13:59:42.108      at Module.require (internal/modules/cjs/loader.js:848:19)
13:59:42.108      at require (internal/modules/cjs/helpers.js:74:18)
13:59:42.108      at requireBuilder (/var/task/sandbox-worker.js:20225:14)
13:59:42.108      at installToolsStep2 (/var/task/sandbox-worker.js:20329:15)
13:59:42.108      at mainSub (/var/task/sandbox-worker.js:20123:33)
13:59:42.108      at async main (/var/task/sandbox-worker.js:20023:5) {
13:59:42.108    code: 'MODULE_NOT_FOUND',
13:59:42.108    path: '/vercel/4c23e98f/.build-utils/.builder/node_modules/@frontity/now/package.json',
13:59:42.108    requestPath: '/vercel/4c23e98f/.build-utils/.builder/node_modules/@frontity/now'
13:59:42.108  }
13:59:44.165  Done with "package.json"
```

--- a/src/index.ts
+++ b/src/index.ts
@@ -154,7 +154,7 @@ export async function build({
     const favicon = await glob("favicon.ico", workPath);
 
     if (robotsTxt)
-      routes.push({ src: `${prefix}/robots.txt`, dest: "robots.txt" });
+      routes.unshift({ src: `${prefix}/robots.txt`, dest: "robots.txt" });
 
     const launcherFiles = {
       "now__bridge.js": new FileFsRef({

--- a/src/index.ts
+++ b/src/index.ts
@@ -153,7 +153,9 @@ export async function build({
     const robotsTxt = await glob("robots.txt", workPath);
     const favicon = await glob("favicon.ico", workPath);
 
-    if (robotsTxt)
+    console.log(JSON.stringify(robotsTxt));
+
+    if (robotsTxt["robots.txt"])
       routes.unshift({ src: `${prefix}/robots.txt`, dest: "robots.txt" });
 
     const launcherFiles = {

--- a/src/index.ts
+++ b/src/index.ts
@@ -12,7 +12,7 @@ import {
   BuildOptions,
   Config,
   FileFsRef,
-  Lambda
+  Lambda,
 } from "@now/build-utils";
 
 interface PackageJson {
@@ -89,7 +89,7 @@ export async function build({
   entrypoint,
   workPath,
   config,
-  meta = {}
+  meta = {},
 }: BuildOptions): Promise<{ routes: object; output: Output }> {
   console.log("Downloading user files...");
   await download(files, workPath, meta);
@@ -117,15 +117,14 @@ export async function build({
       {
         src: `${prefix}/static/(.*)`,
         headers: { "cache-control": "public,max-age=31536000,immutable" },
-        dest: `/static/$1`
+        dest: `/static/$1`,
       },
-      { src: `${prefix}/robots.txt`, dest: "robots.txt" },
       { src: `${prefix}/favicon.ico`, dest: "favicon.ico" },
       {
         src: `${prefix}($|/.*)`,
         headers: { "cache-control": "s-maxage=1,stale-while-revalidate" },
-        dest: `/server.js`
-      }
+        dest: `/server.js`,
+      },
     ];
 
     const nodeVersion = await getNodeVersion(entrypointDir, minNodeRange);
@@ -151,16 +150,19 @@ export async function build({
     validateDistDir(distPath, meta.isDev, config);
     const statics = await glob("static/**", distPath);
     const server = await glob("server.js", distPath);
-    const robotsTxt = await glob('robots.txt', workPath);
+    const robotsTxt = await glob("robots.txt", workPath);
     const favicon = await glob("favicon.ico", workPath);
+
+    if (robotsTxt)
+      routes.push({ src: `${prefix}/robots.txt`, dest: "robots.txt" });
 
     const launcherFiles = {
       "now__bridge.js": new FileFsRef({
-        fsPath: require("@now/node-bridge")
+        fsPath: require("@now/node-bridge"),
       }),
       "now__launcher.js": new FileFsRef({
-        fsPath: path.join(__dirname, "launcher.js")
-      })
+        fsPath: path.join(__dirname, "launcher.js"),
+      }),
     };
 
     const lambda = await createLambda({
@@ -169,16 +171,16 @@ export async function build({
       files: {
         ...launcherFiles,
         "index.js": new FileFsRef({
-          fsPath: server["server.js"].fsPath
-        })
-      }
+          fsPath: server["server.js"].fsPath,
+        }),
+      },
     });
 
     const output = {
       ...statics,
       ...robotsTxt,
       ...favicon,
-      "server.js": lambda
+      "server.js": lambda,
     };
 
     console.log("Finished.");

